model_analyzer/model_manager.py

```diff
@@ -283,7 +292,8 @@ def _get_measurement_if_config_duplicate(self, run_config):
         # check whether perf config string is a key in result dict
         if model_name not in results:
             return False
-        if model_config_name not in results[model_name]:
+        if not self._is_config_in_results(
+                run_config.model_config()._model_config, results[model_name]):
```

Comment:
I think this will not be equal to True even if the model configurations are equal. The reason is that the model configuration stored in the checkpoint contains the model config variant name of the old run. For example if the current model configuration is `model_i1` and the profiled model is `model_i2` and all the other model configuration fields are equal, then these model configs will not be considered equal.

Reply:
Could you please add a test where the profile command is run twice and there are some intersections between the `model_config_parameters` being profiled in each stage. Then, the total number of results during the analyze step should be `measurements_in_stage1 + measurements_in_stage2 - (measurements_in_both_stages)`.

Reply:
we discussed offline, and we're just going to remove the checkpoint directory on a re-profile with override 👍